tests/integration/test_snapshot_option_update.py

```diff
@@ -381,7 +381,7 @@ def test_used(snapshot):
     )
     snapfile_empty = Path("__snapshots__", "empty_snapfile.ambr")
     testdir.makefile(".ambr", **{str(snapfile_empty): ""})
-    result = testdir.runpytest("-v", "--snapshot-update", "test_used.py")
+    result = testdir.runpytest("-v", "test_used.py", "--snapshot-update")
```

Comment:
pytest supports both orders, doesn't it? considering the tests were passing.

Reply:
@noahnu yeah it does but the change fails on the previous code. The previous code was not handling the arguments correctly

Reply:
As long as we have tests for both argument orders, then this LGTM

Reply:
We do, only changed some of them